datasrcs/script_exports.go

```diff
@@ -608,6 +608,17 @@ func (s *Script) scrape(L *lua.LState) int {
 	found = false
 	filter := stringfilter.NewStringFilter()
 	for _, name := range s.subre.FindAllString(resp, -1) {
+		inDomains := false
```

Comment:
You can use the IsDomainInScope method from the config package to implement this check

Reply:
There is a built-in function for this?

Reply:
Yes. WhichDomain returns the domain in scope that the provided FQDN ends with

Reply:
```go
if inDomains := cfg.WhichDomain(name); inDomains != "" {
	continue
}
```

Is that correct?

Reply:
```go
if domain := cfg.WhichDomain(name); domain == name {
continue
}
```

Reply:
Oh yeah, my bad! I misunderstood the function